### ISSUE TITLE
Byte array is mapped to an OpenAPI integer array instead of binary string

### ISSUE THREAD

**Issue by joshuastorch** [javalin/javalin-openapi#58]
**Endpoint:**
```kotlin
@OpenApi(
    methods = [HttpMethod.POST],
    path = "/my/post/endpoint",
    summary = "Post something",
    tags = ["config"],
    requestBody = OpenApiRequestBody(
        content = [OpenApiContent(ByteArray::class)]
    ),
    responses = [
        OpenApiResponse("200"), OpenApiResponse("415")
    ]
)
fun myPostEndpoint(ctx: Context) {
}
```

**Expected output:**
```json
{
  "application/octet-stream": {
    "schema": {
      "type": "string",
      "format": "binary"
    }
  }
}
```
**Actual Output:**
```json
{
  "application/json": {
    "schema": {
      "type": "array",
      "items": {
        "type": "integer",
        "format": "int32"
      }
    }
  }
}
```

**Comment by dzikoysk** [comment]
I guess we should also return such response for `byte[]`, right? I'll try to fix it tomorrow :)

**Comment by joshuastorch** [comment]
I looked a little bit into it and ``ByteArray`` is processed as ``byte[]`` anyway. I implemented a possible solution in #59.
